src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -82,11 +91,53 @@ object WdlExpression {
     val parseTree = parser.parse_e(tokens, new WdlSyntaxErrorFormatter(terminalMap))
     new WdlExpression(parseTree.toAst)
   }
+
+  def toString(ast: AstNode, highlighter: SyntaxHighlighter = NullSyntaxHighlighter): String = {
```

Comment:
Is it expected that one would be bouncing between highlighters in a single JVM? If not, I'm wondering if it'd make sense to have WdlExpression and the `SyntaxFormatter` hierarchy set up so that the final object here contains the appropriate highlighting logic when constructed and then that's just globally used instead of having the `highlighter` argument used throughout.